frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -125,6 +125,7 @@ func httpHandler(h handler) http.Handler {
 			return
 		}
 		replyJSONOK(w, res)
+		log.Println("handling request(%p) ended %s %s", r, r.Method, r.URL.Path)
```

Comment:
move this a `defer` statement below the `log.Println("handling request(%p) started %s %s", r, r.Method, r.URL.Path)`